lib/aptible/api/account.rb

```diff
@@ -31,6 +31,10 @@ class Account < Resource
       field :gentlemanjerry_certificate
       field :gentlemanjerry_docker_name
       field :gentlemanjerry_instance_id
+      field :ca_body
+      field :encrypted_ca_private_key
```

Comment:
This should just be `:ca_private_key`. API itself handles encrypting/decrypting opaquely. For an example of this, you can see Certificates (https://github.com/aptible/aptible-api-ruby/blob/master/lib/aptible/api/certificate.rb#L10), where the same exact thing is done.